README.md

```diff
@@ -1,149 +1,32 @@
-# htbhf-applicant-web-ui
+# Apply for Healthy Start Applicant Web UI
 
-[![Build Status](https://travis-ci.com/DepartmentOfHealth-htbhf/htbhf-applicant-web-ui.svg?branch=master)](https://travis-ci.com/DepartmentOfHealth-htbhf/htbhf-applicant-web-ui)
-[![Known Vulnerabilities](https://snyk.io/test/github/DepartmentOfHealth-htbhf/htbhf-applicant-web-ui/badge.svg?targetFile=package.json)](https://snyk.io/test/github/DepartmentOfHealth-htbhf/htbhf-applicant-web-ui?targetFile=package.json)
-[![License](https://img.shields.io/badge/license-MIT-blue.svg)](https://opensource.org/licenses/MIT)
-[![Coverage Status](https://codecov.io/gh/DepartmentOfHealth-htbhf/htbhf-applicant-web-ui/branch/master/graph/badge.svg)](https://codecov.io/gh/DepartmentOfHealth-htbhf/htbhf-applicant-web-ui)
+A developer's guide for working with the **Apply for Healthy Start** applicant web UI is available in the [project wiki](https://github.com/DepartmentOfHealth-htbhf/htbhf-applicant-web-ui/wiki/Apply-for-Healthy-Start).
```

Comment:
The link on this page goes to a blank page for `Apply-for-Healthy-Start`.

Reply:
Yep - I changed the URL. I will update.